client/k8sextclient/k8sextclient.go

```diff
@@ -1,4 +1,4 @@
-package crdclient
+package k8sextclient
```

Comment:
I named it k8sextclient.K8sExtClient (K8s extensions client), which is now a dependency of CRDClient.